src/main/java/mate/controller/DisplayAllCarsController.java

```diff
@@ -0,0 +1,23 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet ("/cars/cars")
```

Comment:
```suggestion
@WebServlet ("/cars")
```
Fix this in the following cases